pkg/pillar/cmd/downloader/downloader.go

```diff
@@ -372,7 +391,6 @@ func handlersInit() {
 // Determine whether it is an create or modify
 func handleDownloaderModify(ctxArg interface{}, objType string,
 	key string, configArg interface{}) {
-
```

Comment:
FWIW it is easier to read with this empty line to separate the function arguments from the local variables and code, since go's indentation rules don't allow them to be indented differently.